scripts/fetchHarvesterTypes.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable */
+const git = require('nodegit');
+const path = require('path');
+const rimraf = require('rimraf');
+const { exec } = require('child_process');
+
+const BRANCH = process.env.BRANCH;
```

Comment:
This should just be a parameter of the script

Reply:
I dont understand

Reply:
do you mean set it as variable in the code?

Reply:
No, I mean receive it as a parameter, like `yarn my-script some-branch`